src/pages/Home/Home.tsx

```diff
@@ -5,10 +5,14 @@ import FundsList from "../../components/Fund/FundsList";
 import Button from "../../components/Button/Button";
 import FAQ from "../../components/FAQ/FAQ";
 import faqs from "../../config/faqs";
+import Contracts from "../../components/Contracts/Contracts";
+import Payments from "../../components/Payments/Payments";
 
 export default function Home() {
   return (
     <div className="container mx-auto">
+      <Contracts />
```

Comment:
Please move it to Pages/FundYield.tsx. You can replace all the existing code with these 2 components.